src/components/TableParams/Styles.ts

```diff
@@ -0,0 +1,59 @@
+import styled, {css} from 'styled-components';
+import {fontWeightRegular, fontWeightSemiBold} from 'styles/fonts/fontWeights';
+// import colors from 'styles/colors';
+// import type {Shape} from '.';
+
+export const TableParams = styled.div`
+  border: 1px solid #e6e6e6;
+  margin: 0 0 20px;
+  max-width: 100%;
+  overflow-x: auto;
+`;
+
+export const Thead = styled.thead`
+  text-align: left;
+`;
+
+export const Tr = styled.tr`
+  &--even {
+    background-color: #f6f6f6;
+  }
+`;
+
+export const Th = styled.th`
+  padding: 4px 8px;
+`;
+export const Td = styled.td`
+    min-width: 260px;
+    padding: 2px 8px;
+
+    &--param {
```

Comment:
Can you separate as different styled components instead of adding the selectors in?